examples/llm/src/models/mosaic_gpt.py

```diff
@@ -308,6 +308,19 @@ def __init__(self, om_model_config: DictConfig):
             'Perplexity':
                 Perplexity(),
         }
+        if self.hf_config.loss_fn == 'fused_crossentropy':
+            try:
+                from flash_attn.losses.cross_entropy import CrossEntropyLoss as FusedCrossEntropyLoss  # type: ignore # isort: skip
+                print('Using Fused Cross Entropy Loss.')
+                self.loss_fn = FusedCrossEntropyLoss(inplace_backward=False,
+                                                     ignore_index=-100,
+                                                     process_group=None)
+            except:
+                raise ValueError(
+                    'Fused Cross Entropy is not installed. Either (1) have a CUDA-compatible GPU and `pip install .[llm]`, or (2) set your config model.loss_fn=torch_crossentropy.'
+                )
+        else:
```

Comment:
Isn't it more correct to do
`elif self.hf_config.loss_fn == 'torch_crossentropy'`
then `else: raise ValueError(f'specified loss_fn {self.hf_config.loss_fn} not recognized')`